libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,7 +7,7 @@
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
+    <button mat-icon-button matSuffix aria-label="search icon">
```

Comment:
If this an appropriate label? Why would a partially sighted user need to know that this is an icon?

_Is_ it an icon?

Reply:
updated - removed icon text